MatthieuBonne/CosmosQuest-src
# Remove "750UM" button

Goal is to delete 0ghk and change it to text. (This image was ugly anyway)

How to test:
Go to hero manager. search for gaiabyte, dr hawking, spyke or Aoyuki and see the 750 UM in the left bottom corner.
Discard the demo.html changes if needed.

Changes in demo.html are only for testing the 4 um levelable heroes. note: in demo.html it is not possible to actually upgrade heroes. but promotions work. strange behaviour!
